src/main/resources/config.yml

```diff
@@ -1,63 +1,62 @@
 settings:
   draft: draft07
   regions:
-    # US
```

Comment:
[sorted diff](https://www.diffchecker.com/k3f4zruF)

Reply:
@rjlohan because of https://github.com/aws-cloudformation/aws-cloudformation-template-schema/pull/25, tested without `regions:` block and it still seemed to work for me, is this list even needed?

Reply:
Yep this is a default setting that can be overridden.